install_config/aggregate_logging_sizing.adoc

```diff
@@ -162,6 +156,44 @@ size].
 
 xref:../install_config/aggregate_logging.adoc#install-config-aggregate-logging[Learn more about installing EFK].
 
+
+[[install-config-aggregate-logging-sizing-guidelines-large-cluster-installation]]
+=== For large clusters
+
+At 100 nodes or more, it is recommended to pre-pull the logging images first and to set `ImagePullPolicy: IfNotPresent` in the logging-deployer.yaml file. After deploying the logging infrastructure pods (elasticsearch, kibana and curator), node labeling should be done in steps of 20 nodes at a time. For example:
+
+Using a simple loop:
+
+----
+$ while read node; do oc label nodes $node logging-infra-fluentd=true; done < 20_fluentd.lst
+----
+
+The below also works:
+
+----
+$ oc label nodes 10.10.0.{100..119} logging-infra-fluentd=true
+----
+
+This is done in order to avoid a possible thundering herd scenario against your image registry.
+
+
+[NOTE]
+====
+Check for the occurence of any "CrashLoopBackOff | ImagePullFailed | Error" issues.
+`oc logs <pod>`, `oc describe pod <pod>` and `oc get event` are helpful diagnostic commands.
+====
+
+[[install-config-aggregate-logging-sizing-guidelines-split-fluentd-install]]
+=== Split fluentd
+
+Refer to: 
+link:https://github.com/ewolinetz/origin-aggregated-logging/blob/7a814f58513b03f1660290864e550efc9db1e7a6/deployer/README.md#splitting-your-fluentd-for-high-scale-capacity-advanced[Split Fluentd]
```

Comment:
Personally I would rather this was a KBase article since it should eventually be phased out for a better approach on how Fluentd will collect metadata from OCP (i.e. querying kubelets over the master api)